CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 7.0.1
+
+* Add an `extra_require` to `setup.py` to express a dependency on `OpenFisca-Parsers`.
```

Comment:
* Declare `Openfisca-Parsers` as an optional dependency of `Openfisca-Core`

(less technical, more functional)